.github/workflows/self-hosted-gpu-test.yml

```diff
@@ -0,0 +1,126 @@
+name: self-hosted-gpu-test
+on:
+  push:
+    branches:
+      - master
+      - feat/add_aws_testing 
```

Comment:
This should not be here, right?

Reply:
Yes, I will remove that before we merge it. For testing/making this PR, this makes it easy to test.